### PR TITLE
Add MonetaryUnit (MUE) support

### PR DESCRIPTION
**Pull request by sotblad**

### CODE REVIEW

**Comment by sotblad** [comment]
Tested and it's working!
Screenshot 2020-08-03 at 6 39 40 PM

File: NBXplorer.Client/NBXplorer.Client.csproj [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
 		<TargetFramework>netstandard2.0</TargetFramework>
 		<Company>Digital Garage</Company>
-		<Version>3.0.15</Version>
+		<Version>3.0.17</Version>
```
Comment (NicolasDorier) [inline NBXplorer.Client/NBXplorer.Client.csproj]: don't bump this

**Comment by sotblad** [comment]
Hey @NicolasDorier, sorry for disturbing you but I would like to ask, when approximately will you bump NBXplorer to v3.0.18? Thanks!

**Comment by NicolasDorier** [comment]
done

**Pull request merged**